packages/calcite-components/src/components/table-cell/resources.ts

```diff
@@ -5,4 +5,5 @@ export const CSS = {
   selectedCell: "selected-cell",
   assistiveText: "assistive-text",
   lastCell: "last-cell",
+  nonInteractive: "non-interactive",
```

Comment:
Nitpick: should this and related styles reflect the updated name?

Reply:
Updated to match other css cell modifiers 👍